modules/ols-0-2-1-release-notes.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+// release_notes/ols-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ols-0-2-1-release-notes_{context}"]
+= {ols-long} version 0.2.1
+
+{ols-official} 0.2.1 is now available on {ocp-product-title} 4.15 or later.
+
+[id="ols-0-2-1-enhancements_{context}"]
+== Enhancements
+
+The following enhancements have been made with {ols-official} 0.2.1:
+
+* With this release, the {ols-long} User Interface is hidden if the *Hide Lightspeed* option is selected in *User Preferences*.
+** Xavier to review in latest product build
+
+* The text area of the {ols-long} User Interface now generates a preview when you attach an event to a prompt. The preview presents the actual YAML text for the event, helping you determine if attaching the event will provide additional context for the question being asked.
+
+* This release adds the ability to attach a YAML file to a question you ask on any page accessible from *Virtualization* in the {ocp-product-title} web console. This release also adds the ability to attach events and logs for `VirtualMachines` and `VirtualMachineInstances`.
+
+* Beginning in this release you can set the `api_version` parameter for the {azure-openai} provider in the `OLSConfig` Custom Resource.
+** OLS-1092
+
+* This release provides the ability to make use of the cluster-wide Transport Layer Security (TLS) profile and to make use of the TLS custom profile for the {ols-long} stack.
+** OLS-1097
+
+* This release also adds the ability to use user provided TLS certificates for encrypting communication between {ols-long} services.
```

Comment:
This release adds ability for the user to provide their own TLS certificates for the communication between console proxy and the application HTTPS endpoint.